src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -35,16 +35,43 @@ public RemoteControlServer(IConfiguration configuration)
 					this.Log().LogDebug("Unloading assembly context");
 				}
 			};
+
+			// Add custom resolving so we can find dependencies even when the processor assembly
+			// is built for a different .net version than the host process.
 			_loadContext.Resolving += (context, assemblyName) =>
 			{
 				if (_loopAssembly is not null)
 				{
-					return context.LoadFromAssemblyPath(Path.Combine(Path.GetDirectoryName(_loopAssembly.Location)!, assemblyName.Name + ".dll"));
-				}
-				else
-				{
-					return context.LoadFromAssemblyName(assemblyName);
+					try
+					{
+						var loc = _loopAssembly.Location;
+
+						if (!string.IsNullOrWhiteSpace(loc))
```

Comment:
Each of these "if" conditions need an else, if only for outputting a meaningful log.
Also, if the processor doesn't load, what's the fallback - is there a way the user can fix this?

Reply:
added logging in an else clause

Reply:
If a processor doesn't load it's likely down to be trying to use something unsupported or a processor that isn't packed/distributed correctly. 
In either case, the fix isn't going to be straight forward and not something an end developer can deal with beyond reporting with full logs.